docs_guide/intro/how_to_use_this_guide.rst

```diff
@@ -23,4 +23,12 @@ Similarly, the :ref:`overview`, :ref:`getting_started` and :ref:`release_notes`
 
 If what you are looking for is not how to deal with the content of your documentation but with the more technical details of creating documentation with :doc:`Sphinx <sphinx:index>`, the documentation builder we are using in Qiskit,
 check the :ref:`sphinx_guide` section. There you will find explanatory material about Sphinx and ReStructuredText (Sphinx's native language), as well as guides for Sphinx-related tasks such as
-building documentation, structuring it with different headers and lists and including links and technical expressions like code blocks or mathematic formulae.
\ No newline at end of file
+building documentation, structuring it with different headers and lists and including links and technical expressions like code blocks or mathematic formulae.
+
+
+Contributing to this guide
+==========================
+
+If you want to help improve this guide, you can open an issue in the [qiskit_sphinx_theme](https://github.com/Qiskit/qiskit_sphinx_theme) repository with the label
+`qiskit docs guide` to make any suggestions or directly create a pull request to the `feature/docs-guidelines` `branch <https://github.com/Qiskit/qiskit_sphinx_theme/tree/feature/docs-guidelines>`_
```

Comment:
People generally won't be able to label because they don't have triage permission of the repo. We should create an issue template for that. For that I created an issue to track: https://github.com/Qiskit/qiskit_sphinx_theme/issues/172. The `feature/docs-guidelines` branch should be merged to `main` next week and will be removed. So I think you can just mention make changes to `doc_guide` folder and link to https://github.com/Qiskit/qiskit_sphinx_theme